src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,445 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Customize web media notifications and respond to media related events with the new Media Session API.
+
+{# wf_updated_on: 2017-02-06 #}
+{# wf_published_on: 2017-02-06 #}
+{# wf_tags: news,chrome57,media,notifications,play #}
+{# wf_featured_image: /web/updates/images/2017/02/tldr.png #}
+{# wf_featured_snippet: Finally! We can customize web media notifications (title, artist, album name, artwork) and respond to media related events such as seeking or track changing with the new Media Session API. #}
+
+# Customize Media Notifications & Handle Playlists {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+With the brand new [Media Session API], you can now **customize media
+notifications** by providing metadata information of the media your web app is
+playing in Chrome 57 (beta in February 2017, stable in March 2017). It also
```

Comment:
I think you should probably add a full stop after 'playing' and then start a new sentence with '`MediaSession` is supported in Chrome 57 ...'